src/parser/jobs/mnk/modules/evaluators/BrotherhoodEvaluator.tsx

```diff
@@ -1,98 +1,105 @@
+import {EvaluatedAction} from 'parser/core/modules/ActionWindow'
+import {RulePassedEvaluator} from 'parser/core/modules/ActionWindow/evaluators/RulePassedEvaluator'
 import {HistoryEntry} from 'parser/core/modules/ActionWindow/History'
-import {SEVERITY, Suggestion} from 'parser/core/modules/Suggestions'
+import {SEVERITY, SeverityTiers, Suggestion, TieredSuggestion} from 'parser/core/modules/Suggestions'
 import React from 'react'
-import {Brotherhood, OverlapStatus} from '../Brotherhood'
+import {OverlapStatus} from '../Brotherhood'
 
-export class BrotherhoodDriftEvaluator extends NotesEvaluator {
-	private brotherhood!: Brotherhood
+export interface BrotherhoodEvaluatorOpts {
+	suggestionIcon: string
+	getOverlapStatus: (window: HistoryEntry<EvaluatedAction[]>) => OverlapStatus
+}
+
+export class BrotherhoodDriftEvaluator extends RulePassedEvaluator {
 	private suggestionIcon: string
+	private getOverlapStatus: (window: HistoryEntry<EvaluatedAction[]>) => OverlapStatus
 
-	constructor(brotherhood: Brotherhood, suggestionIcon: string) {
+	constructor(opts: BrotherhoodEvaluatorOpts) {
 		super()
-		this.brotherhood = brotherhood
-		this.suggestionIcon = suggestionIcon
+		this.suggestionIcon = opts.suggestionIcon
+		this.getOverlapStatus = opts.getOverlapStatus
 	}
 
 	header = {
-		header: <DataLink showName={false} action="BROTHERHOOD"/>,
+		header: <Trans id="mnk.brotherhood.drift.header">Used inside<br/><DataLink action="RIDDLE_OF_FIRE" showIcon={false}/>?</Trans>,
 		accessor: 'brotherhoodstatus',
 	}
 
-	// For any given RoF window, a brotherhood window can
-	// * Start before and end during (Used early)
-	// * Start during and end during (Correct)
-	// * Start during and end after (Used late)
-	// * Be completely unrelated
-	override generateNotes(window: HistoryEntry<EvaluatedAction[]>) {
-		const matchedWindow = this.brotherhood.findMatchingWindow(window)
-		if (matchedWindow == null) {
-			return <Trans id="mnk.rof.chart.notes.none">None</Trans>
+	override suggest(windows: Array<HistoryEntry<EvaluatedAction[]>>): Suggestion | undefined {
+		const windowStatuses = windows.map(window => this.getOverlapStatus(window))
+		const driftedWindows = windowStatuses.filter(w => w === OverlapStatus.USED_EARLY || w === OverlapStatus.USED_LATE)
+		if (driftedWindows.length === 0) {
+			return undefined
 		}
 
-		switch (matchedWindow.status) {
+		return new Suggestion({
+			icon: this.suggestionIcon,
+			content: <Trans id="mnk.brotherhood.drifted-window.content.">
+				Try and make sure your <DataLink action="BROTHERHOOD"/> casts are fully included within a <DataLink action="RIDDLE_OF_FIRE"/> window.
+			</Trans>,
+			severity: SEVERITY.MINOR,
+			why: <Trans id="mnk.brotherhood.drifted-window.why">
+				<Plural value={driftedWindows.length} one= "# cast" other="% casts"/> of <DataLink action="BROTHERHOOD"/> was either early or late for a <DataLink action="RIDDLE_OF_FIRE"/> window.
```

Comment:
```suggestion
				<Plural value={driftedWindows.length} one="# cast" other="# casts"/> of <DataLink action="BROTHERHOOD"/> <Plural value={driftedWindows.length} one="was" other="were"/> either early or late for a <DataLink action="RIDDLE_OF_FIRE"/> window.
```

Template error for the plural value, holdover from production.  Might be worth fixing the verb while we're mucking with it?